rest-api-spec/src/main/resources/rest-api-spec/api/cluster.get_decommission_awareness.json

```diff
@@ -0,0 +1,29 @@
+{
+  "cluster.get_decommission_awareness": {
+    "documentation": {
+      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/decommission/",
+      "description": "Get details and status of decommissioned attribute"
+    },
+    "stability": "experimental",
```

Comment:
Why is this experimental? Do we think the API can be changed in the future?

Reply:
Marked it experimental at the moment as its hooks, etc are still under development. Once all related changes including recommission/decommission/controlled discovery/kill switch are merged will raise a PR to mark this `stable`.
Meta issues related to project can be found here - https://github.com/opensearch-project/OpenSearch/issues/3957